clients/driver/driver.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+require('dotenv').config();
+//connect to CAPS as client
+const IO = require('socket.io-client');
+const HOST = process.env.HOST;
+const CAPS = IO.connect(`${HOST}/caps`);
+const CONN = IO.connect(HOST);
+
+CONN.on('pickup-ready', timer);
+
+function timer() {
+  setTimeout(() => {
+    CAPS.emit('in-transit', order);
+  }, 1000);
+  setTimeout(() => {
+    CAPS.emit('success', order);
+  }, 3000);
+};
```

Comment:
When I try and start this application up I immediately get a message that says: `CAPS WORKING` the problem is that it closes right after that so before I can even get vendor and driver up it closes so no delivery and pickup messages are showing up.